config/deploy.rb

```diff
@@ -44,3 +44,7 @@
 
 # update shared_configs before restarting app
 before 'deploy:restart', 'shared_configs:update'
+
+# configure capistrano-rails to work without sprockets
```

Comment:
This is only necessary for the first deploy right? How do we ensure it gets turned off after?